qa/workunits/cephtool/test.sh

```diff
@@ -1084,11 +1084,29 @@ function test_mon_osd()
   for f in noup nodown noin noout noscrub nodeep-scrub nobackfill norebalance norecover notieragent full sortbitwise
   do
     ceph osd set $f
+    if [[ "$f" == "sortbitwise" ]]; then 
+      expect_failure $TMPDIR "sortbitwise flag is already set"
+    fi
     ceph osd unset $f
+    if [[ "$f" == "full" ]]; then
+      # As full flag gets unset if there wont be any OSD in cluster which is full
+      # Not taking failure as it gets unset by default  
+      expect_failure $TMPDIR "full flag is not set" || return 0
+    fi
   done
   ceph osd set sortbitwise  # new backends cant handle nibblewise
+  expect_failure $TMPDIR "sortbitwise flag is already set"
   expect_false ceph osd set bogus
   expect_false ceph osd unset bogus
+  
+  ceph osd set noout >& $TMPFILE || return 1
+  check_response "set noout"
+  ceph osd set noout >& $TMPFILE || return 1
+  expect_failure $TMPDIR "noout flag is already set"
```

Comment:
@kchai ahh sorry , right it should be `check_response()` instead of `expect_failure()`.
Right, as sage pointed earlier: 

> The mon commands must be idempotent

I think it will apply to all the new tests for new messages ? will change it or do you think any other change is needed.